core/rest-client/src/http_client.rs

```diff
@@ -444,54 +444,92 @@ mod tests {
 	}
 
 	#[test]
+	//This test run because the request throws an error "Resource temporarily unavailable" and not because the
```

Comment:
I have some trouble understanding this sentence. Did you mean:

```suggestion
	// FIXME: This test passes because the request throws an error "Resource temporarily unavailable" due to a timeout error.
	// Technically, it should check for a blocked, valid `letsencrypt_isrgrootx1` certificate.
```

Reply:
Maybe you should comment these tests? I don't think passing tests, that should not actually pass, are the way to go.

Reply:
I wrote those test to understand/check how we handle Let's Encrypt certificates. I update the comments to make it clearer, but I think they are useful and don't want to remove them. 
Adding a FIXME is a good point.

Reply:
Never meant to remove the comments, but rather to comment the tests, because currently they are not testing what the name is indicating.